src/test/resources/org/broadinstitute/hellbender/tools/haplotypecaller/expected.testGVCFMode.gatk4.alleleSpecific.g.vcf

```diff
@@ -240,7 +240,7 @@
 20	10005724	.	A	<NON_REF>	.	.	END=10006290	GT:DP:GQ:MIN_DP:PL	0/0:82:99:53:0,120,1800
 20	10006291	.	G	A,<NON_REF>	999.64	.	AS_RAW_BaseQRankSum=|0.3,1|NaN;AS_RAW_MQ=188569.00|106610.00|0.00;AS_RAW_MQRankSum=|-1.1,1|NaN;AS_RAW_ReadPosRankSum=|-1.5,1|NaN;AS_SB_TABLE=25,28|16,15|0,0;BaseQRankSum=0.311;DP=85;ExcessHet=3.0103;MLEAC=1,0;MLEAF=0.500,0.00;MQRankSum=-1.081;RAW_MQandDP=298779,85;ReadPosRankSum=-1.447	GT:AD:DP:GQ:PL:SB	0/1:53,31,0:84:99:1007,0,1905,1167,1998,3165:25,28,16,15
 20	10006292	.	A	<NON_REF>	.	.	END=10006403	GT:DP:GQ:MIN_DP:PL	0/0:81:99:76:0,120,1800
-20	10006404	.	A	C,<NON_REF>	2943.06	.	AS_RAW_BaseQRankSum=||;AS_RAW_MQ=0.00|268082.00|1369.00;AS_RAW_MQRankSum=||;AS_RAW_ReadPosRankSum=||;AS_SB_TABLE=0,0|27,49|1,0;DP=83;ExcessHet=3.0103;MLEAC=2,0;MLEAF=1.00,0.00;RAW_MQandDP=291051,83	GT:AD:DP:GQ:PL:SB	1/1:0,76,1:77:99:2957,228,0,2957,228,2957:0,0,28,49
```

Comment:
This is an interesting lateral effect fix..... there were some reads that supported the NON_REF that should not be there as per policy .... that was result of this very same bug as filtered reads were supporting the NON_REF just at random. 

So what we should see in this this test vcf updates is some (usually) small changes in AD and annotations that depend on read likelihoods across all alleles in general and specifically for NON_REF where spurious "supporting reads" will disappear (which is the right thing).